docs/authoring.md

```diff
@@ -86,78 +86,132 @@ To make your component available as a NuGet package, it is important to include
 
 This section describes the steps needed to consume a C#/WinRT component for the following scenarios:
 
-- [C++/WinRT desktop applications (Unpackaged)](#consuming-from-cwinrt-unpackaged)
-- [C# desktop applications](#consuming-from-c-applications) (.NET 5 or later)
-- [Packaged applications](#Consuming-from-packaged-applications)
-- [Out of process components](#Consuming-an-out-of-process-component)
+- [C# desktop applications](#consuming-from-c-applications) 
+- [C++ desktop applications](#consuming-from-c-applications)
+- [Windows App SDK applications](#windows-app-sdk-applications)
+- [Out of process components](#consuming-an-out-of-process-component)
 
-### Consuming from C++/WinRT (Unpackaged)
+### Consuming from C# applications
+
+Consuming a C#/WinRT component from C#/.NET apps is supported by both package reference or project reference. This scenario is equivalent to consuming any ordinary C# class library and does not involve WinRT activation in most cases. This scenario is fully supported on .NET 6 and later.

```

Comment:
in most cases -> can we be more precise? what are the scenarios where it does involve winrt activation?

Reply:
IIUC, We are referring to out of process support, a few sections down, where you need a runtimeconfig.json. There are no scenarios where you need a manifest file for c#. Correct me if I'm wrong @angelazhangmsft  :)

I'll make the clarification in the docs